bgp/modules/terms.py

```diff
@@ -113,3 +115,37 @@ def validate_isbn(isbn):
     def __init__(self):
         super(IsbnExtractorModule, self).__init__(self.validate_isbn)
 
+
+class PageTypeProcessor():
+
+    def __init__(self, modules):
+        """
+        :param modules: a dict of {'name': module}
+        """
+        self.modules = modules
+
+class CopyrightPageDetectorModule:
```

Comment:
The CopyrightPageDetectorModule should inherit the base class which in this case would be PageTypeProcessor.